src/components/SideBar/SideBar.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import {
+  Divider,
+  RadioGroup,
+  Radio,
+  FormControl,
+  FormLabel,
+  FormControlLabel,
+  FormGroup,
+  Checkbox,
+} from '@material-ui/core';
+import { useStyles } from './SideBar.styles';
+import useTexts from '../../hooks/useTexts';
+
+const SideBar: React.FC = () => {
+  const texts = useTexts();
+  const classes = useStyles();
+  const [value, setValue] = React.useState('subcategory');
+  const [state, setState] = React.useState({
```

Comment:
state, setState => what state? Try to use more descriptive names (e.g checkboxState, setCheckboxState).